index.bs

```diff
@@ -6024,6 +6159,11 @@ partial dictionary MLOpSupportLimits {
     <td>specified as part of operation steps</td>
     <td>4</td>
```

Comment:
(tangential) I'd like to relax this some, since all backends support at least 1-4. Planning to open issue. ⏳